lib/src/options.dart

```diff
@@ -105,15 +105,25 @@ class VideoPublishOptions {
       '${runtimeType}(videoEncoding: ${videoEncoding}, simulcast: ${simulcast})';
 }
 
+class AudioPreset {
+  static const telephone = 12000;
+  static const speech = 20000;
+  static const music = 32000;
+}
+
 /// Options used when publishing audio.
 class AudioPublishOptions {
   /// Whether to enable DTX (Discontinuous Transmission) or not.
   /// https://en.wikipedia.org/wiki/Discontinuous_transmission
   /// Defaults to true.
   final bool dtx;
 
+  /// max audio bitrate
+  final int audioBitrate;
```

Comment:
Maybe this can be simply `bitrate` ? since it's already in `AudioPublishOptions` ?

Reply:
Makes sense, will update in a while. 
maybe it can be called `maxBitrate`, which is a limit parameter.

Reply:
makes sense !